src/ucs/sys/preprocessor.h

```diff
@@ -22,11 +26,11 @@
 #endif
 
 /* Creating unique identifiers, used for macros */
-#define UCS_PP_APPEND_UNIQUE_ID(x)      UCS_PP_TOKENPASTE(x, UCS_PP_UNIQUE_ID)
+#define UCS_PP_APPEND_UNIQUE_ID(x)        UCS_PP_TOKENPASTE(x, UCS_PP_UNIQUE_ID)
 
 /* Convert to string */
-#define _UCS_PP_MAKE_STRING(x) #x
-#define UCS_PP_MAKE_STRING(x) _UCS_PP_MAKE_STRING(x)
+#define _UCS_PP_MAKE_STRING(x)            #x
```

Comment:
is there a difference between `# X` and `#X`?

Reply:
AFAIK no, seems we could unite UCS_PP_MAKE_STRING and UCS_PP_QUOTE, but in any case this PR just changed their indentation..

Reply:
thanks for explanation